YamlFile.js

```diff
@@ -142,6 +142,11 @@ var badEndPunct = {
 YamlFile.prototype._isTranslatable = function(resource) {
     var locale = new Locale(this.getLocale());
 
+    var yamlSettings = this.project.settings.yaml;
```

Comment:
As discussed on call with @wadimw, lack of yaml settings leads to 'no action' of loctool in term of string extraction from yaml files (ss attached). So, it will no result in an error here. Maybe we need to mention in readme that yaml settings are required 🤔.
<img width="1265" alt="Screenshot 2023-08-03 at 14 52 54" src="https://github.com/iLib-js/ilib-loctool-yaml/assets/43447293/e47ac23a-23af-4daf-b529-2067b0a2df82">

Reply:
Still, it should not crash even if it does not work properly. We should have
`var yamlSettings = this.project.settings && this.project.settings.yaml;`